client/src/helpers/fhir/fhir.js

```diff
@@ -340,7 +346,7 @@ export const createPatientSubmissionBundle = ({
 
   if (clinicalImpression != null) {
     const clinicalImpressionResource = FhirDataManager.createClinicalImpression(
-      'PR00101', // TODO: Change to real id once it's supported.
+      'PR0001', // TODO: Change to real id once it's supported.
```

Comment:
Should we change this for practitionerId like above?

Reply:
No, it's not the same resource. ClinicalImpression.assessor refers to the resource performing the assessment, it doesn't have to be the practitioner